html2asketch/nodeToSketchLayers.js

```diff
@@ -322,7 +322,18 @@ export default function nodeToSketchLayers(node, options) {
 
       const textValue = fixWhiteSpace(textNode.nodeValue, whiteSpace);
 
+      let id;
+
+      if (textNode.parentNode && textNode.parentNode.dataset) {
+        const {sketchId, sketchText} = textNode.parentNode.dataset;
+
+        if (sketchId || sketchText) {
+          id = `text:${sketchId || sketchText}`;
+        }
+      }
```

Comment:
you are right, this doesn't belong here - html-sketchapp tries to convert HTML/CSS to Sketch w/o any additional hints. This should definitely go into html-sketchapp-cli 👍

Reply:
I've addressed this by adding an `options.onTextGenerate` middleware callback function, which gives access to `textNode.parentNode` to allow for checking DOM attributes in `html-sketchapp-cli`.